test/hypretests_mpi.jl

```diff
@@ -0,0 +1,72 @@
+using HYPRE
```

Comment:
Is there no way to do MPI CI?

Reply:
This is doing MPI CI; this file is executed here: https://github.com/SciML/LinearSolve.jl/blob/fe4cba4de23ea5656bd5d0fd02ca618f12bfa85b/test/hypretests.jl#L166-L170